extensions/sql-database-projects/src/test/baselines/openSdkStyleSqlProjectWithFilesSpecifiedBaseline.xml

```diff
@@ -13,6 +13,7 @@
   <ItemGroup>
     <Folder Include="Properties" />
     <Folder Include="folder1" />
+    <Folder Include="folder2\" />
```

Comment:
What if user has a forward slash instead? (for non-Windows scenarios)

Reply:
only back slashes are supported in sqlproj. Even on mac and linux, we write file paths with backslashes so that the same project will work cross-plat and with VS.